go/response_handlers.go

```diff
@@ -215,6 +235,100 @@ func parseSet(response *C.struct_CommandResponse) (any, error) {
 	return slice, nil
 }
 
+// parseLCSMatchedPositions converts the nested array structure from LCSWithOptions into a slice of LCSMatchedPosition structs
+// The input structure has the shape of:
+// ```
+// 1) 1) 1) (integer) 4
+//  2. (integer) 7
+//  2. 1) (integer) 5
+//  2. (integer) 8
+//  2. 1) 1) (integer) 2
+//  2. (integer) 3
+//  2. 1) (integer) 0
+//  2. (integer) 1
+//
+// ```
+// which represents matched positions between two strings
+func parseLCSMatchedPositions(matches any) ([]models.LCSMatchedPosition, error) {
```

Comment:
Move to `converter_helpers.go` please, we can re-use it for a cmd and for batch then.